tas-env-variables.sh

```diff
@@ -1,25 +1,19 @@
 #!/bin/bash
 
-export BASE_HOSTNAME=apps.$(oc get dns cluster -o jsonpath='{ .spec.baseDomain }')
-echo "base hostname = $BASE_HOSTNAME"
+# This assumes you are currently running from the context of the namespace where your securesign is created
+# Run `oc project <securesign namespace>` to ensure you are working within the correct context
```

Comment:
this should accept an env var for the securesign namespace but I'm being lazy here and maybe can be done in a follow-up